nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -1273,6 +1273,43 @@ public void overrideFailsOnExplicitlyNullableLibraryModelParam() {
         .doTest();
   }
 
+  @Test
+  public void overrideOnExplicitlyNonNullLibraryModelReturn() {
+    compilationHelper
+        .addSourceLines( // Dummy android.support.v4.app.Fragment class
+            "Fragment.java",
+            "package android.support.v4.app;",
+            "public interface Fragment {",
+            // Ignore other methods for this test, to make code shorter on both files:
+            "    boolean getActivity();",
+            "}")
+        .addSourceLines(
+            "TestNegative.java",
+            "package com.uber;",
+            "import android.support.v4.app.Fragment;",
+            "class TestNegative implements Fragment {",
+            "  TestNegative() {  }",
+            "  @Override",
+            "  public boolean getActivity() {",
+            "    return false; // NoOp",
+            "  }",
+            "}")
+        .addSourceLines(
+            "TestPositive.java",
+            "package com.uber;",
+            "import javax.annotation.Nullable;",
+            "import android.support.v4.app.Fragment;",
+            "class TestPositive implements Fragment {",
+            "  TestPositive() {  }",
+            "  @Override",
+            "  // BUG: Diagnostic contains: method returns @Nullable",
+            "  @Nullable public boolean getActivity() {",
```

Comment:
It's strange for these methods to have `boolean` type, since `boolean` can never be `null`.  Can we rewrite the test to use a reference type?